src/DynamoUtilities/PathHelper.cs

```diff
@@ -107,10 +107,10 @@ public static bool isValidXML(string path, out XmlDocument xmlDoc)
                 xmlDoc.Load(path);
                 return true;
             }
-            catch
+            catch(Exception ex)
             {
                 xmlDoc = null;
-                return false;
+                throw ex;
```

Comment:
This behavior is different from before, is this on purpose? (i.e. before a false would have been returned, and now an exception will be thrown.)

Reply:
This is on purpose so that DynamoModel can catch exception and determine what to do based on exception type